libguard/guard_interface.cpp

```diff
@@ -75,7 +77,8 @@ const fs::path& getGuardFilePath()
 {
     if (guardFilePath.empty())
     {
-        throw std::runtime_error(
+        guard_log(GUARD_ERROR, "Guard file is not initialised.");
```

Comment:
Do you think adding `"Please make sure libguard_init() is called already"` will help it more from debugging point of view?

Reply:
Not really as the user can get the exception. Also with trace itself we can makeout that file was not initialized.